tests/gen/gen_auto_test.go

```diff
@@ -1352,7 +1352,7 @@ func TestAutoGenerate_IfColDefinitionsAreValid_ShouldGenerate(t *testing.T) {
 						Kind: client.FieldKind_STRING,
 					},
 					{
-						Name:   "owner",
+						Name:   "owner_id",
```

Comment:
question: This change seems wrong, and I can't spot anything in the PR description.  Why has this changed?

Reply:
internally the relationship is set on `owner_id` and not `owner`. For this test to work properly it has to be `owner_id`.

Reply:
I am a bit confused about this one too

Reply:
When we define a schema, in the SDL only `owner` would be defined but internally we also define `owner_id` and this is the field that hold the foreign docID. For this test, the code will look specifically for `owner_id` to set the foreign docID to. It previously worked before but was accidental and probably a bug.

Reply:
I don't think this is correct though - `Kind` is `client.FieldKind_FOREIGN_OBJECT`, but `foo_id` fields should be of kind `INTERNAL_ID` IIRC

Reply:
I fixed it to be representative of what it would be. 
```go
{
	Name:         "owner_id",
	Kind:         client.FieldKind_DocKey,
	RelationType: client.Relation_Type_INTERNAL_ID,
}
```